guix-science/packages/zotero.scm

```diff
@@ -0,0 +1,63 @@
+(define-module (guix-science packages zotero)
+  #:use-module (guix packages)
+  #:use-module ((guix licenses) #:prefix license:)
+  #:use-module (gnu packages base)
+  #:use-module (guix download)
+  #:use-module (nonguix build-system binary)
```

Comment:
We cannot depend on nonguix here.  The `guix-science` channel should not depend on the `nonguix` channel.

Reply:
Well since I have to rely on the binary to build it, and guix itself doesn't have a binary build-system, it's all I've got to work with. See the following mailing list archive: https://lists.gnu.org/archive/html/guix-devel/2019-03/msg00496.html

Additionally, this channel is dedicated to providing open source scientific software; which Zotero is. If the issue is with the build-system and the use of the blob, can we just move this to guix-science nonfree which also uses blobs? 

Building this from source, while preferable, takes a lot of time (as you're basically just building a modified version of old Firefox with extra features), and since we don't have access to Cuirass to allow for substitutes, it really seems like this is the easiest option for users and packagers, especially since the current alternative for guix users it to just use the tarball anyway.